frontend/src/static/js/v_zoom.js

```diff
@@ -145,6 +153,22 @@ window.vZoom = {
       this.isFileTypesLoaded = true;
     },
 
+    retrieveHashes() {
+      this.retrieveSortHash();
+      this.retrieveSelectedFileTypesHash();
+    },
+
+    retrieveSortHash() {
+      window.decodeHash();
+      const hash = window.hashParams;
+      if (hash.zCST) {
+        this.commitsSortType = hash.zCST;
+      }
+      if (hash.zRSC) {
+        this.toReverseSortedCommits = (hash.zRSC === 'true');
```

Comment:
Why do we need the `hash.zRSC === 'true'` ?

Reply:
hashes are stored as a string, I did this to convert the string into a boolean